src/logging/archivator.hpp

```diff
@@ -0,0 +1,68 @@
+#ifndef ARCHIVATOR_HPP_
+#define ARCHIVATOR_HPP_
+
+#include <string>
+
+namespace aos {
```

Comment:
We should use aos::<application name> namespace pattern for applications
```suggestion
namespace aos {
namespace sm {
```

Reply:
done